toolset/setup/linux/webservers/mongrel2.sh

```diff
@@ -23,7 +22,14 @@ fw_get https://raw.github.com/zedshaw/mongrel2/9b565eeea003783c47502c2d350b99c96
 mv -f zmq_compat.h mongrel2-install/src/
 
 cd mongrel2-install
-PREFIX=${IROOT}/mongrel2 make clean all
-PREFIX=${IROOT}/mongrel2 make install
 
-touch ${IROOT}/mongrel2.installed
\ No newline at end of file
+# Do this in a subshell to avoid leaking env variables
```

Comment:
This technique is interesting, but ultimately meaningless in the context of this project - the `testrunner` user that invokes this script is run from a brand new shell, so we don't care if we leak environment variables into that shell. Specifically, once the test is completed, we nuke that shell anyway.